client/src/components/NewOpportunityForm.js

```diff
@@ -31,9 +32,10 @@ class NewOpportunityForm extends Component {
     cities: [],
     skills: [],
     success: null,
-    message: {}
+    open: false
```

Comment:
could we give this a more explicit name please, to indicate what exactly it's tracking? For example `showFormSubmissionOutcome`